Kubernetes/HOWTO-on-prem.md

```diff
@@ -86,37 +118,53 @@ Run the script `start-kubelet.sh`. You may need to run it with `sudo`. Then, in
 
     ~/kube $ sudo cp ~/.kube/config kubelet/
 
-**TODO**: Why is this necessary? Shouldn't this be done automatically by one of the containers?
+**TODO**: Why is this necessary? Shouldn't this be done automatically by one of the containers? **What does this do?**
 
 In another terminal session, run the Kubeproxy script, passing your cluster CIDR (you may also need to run this one with `sudo`):
 
     ~/kube $ ./kubeproxy.sh 192.168
 
 After a few minutes, you should see the following system state:
 
-  - Under `docker ps`, you should see worker and pod containers mirroring the configs in `manifest/` and in `addons/`.
-  - A call to `~/kube/bin/kubectl cluster-info` should show info for the Kubernetes master API server, plus the DNS and heapster addons.
-  - A new interface `cbr0` under `ifconfig`, with your chosen cluster CIDR.
+- Under `docker ps`, you should see worker and pod containers mirroring the configs in `manifest/` and in `addons/`.
+- A call to `~/kube/bin/kubectl cluster-info` should show info for the Kubernetes master API server, plus the DNS and heapster addons.
+- A new interface `cbr0` under `ifconfig`, with your chosen cluster CIDR.
+
+## Prepare a Windows Node ##
+
+We need a baseline of configuration on Windows nodes. This assumes Windows Server 2016, RS3+. This can be in a hypervisor or otherwise, but the instances require a external network IP (accessible by other hosts, not the public internet).
+
+The following is for current insider builds, at an *elevated* Powershell prompt
+
+```powershell
+Install-Module -Name DockerMsftProviderInsider -Repository PSGallery -Force
+Install-Package -Name docker -ProviderName DockerMsftProviderInsider -RequiredVersion preview
+Restart-Computer -Force
+```

```

Comment:
Awesome, thanks for this as well. I am in the process of setting up a Windows node from an "insider" perspective as well to get some more details on this.